plugins/ftdidmx/FtdiDmxPort.h

```diff
@@ -39,13 +43,14 @@ namespace ola {
 namespace plugin {
 namespace ftdidmx {
 
-class FtdiDmxOutputPort : public ola::BasicOutputPort {
+class FtdiDmxOutputPort
```

Comment:
Was this auto-formatting by the IDE? Can you remove it please? We've got some suitable editor configs: https://github.com/OpenLightingProject/editor-configs

Can you either use one of those, or tune yours and submit it if you're using an alternative editor.

Reply:
No, at some point I inherited 2 classes so it was better looking on the next line.

I'll revert that.

Reply:
Thanks